test/e2e/proxy.go

```diff
@@ -65,7 +65,10 @@ func proxyContext(version string) {
 	It("should proxy logs on node using proxy subresource [Conformance]", func() { nodeProxyTest(f, prefix+"/nodes/", "/proxy/logs/") })
 	It("should proxy to cadvisor using proxy subresource [Conformance]", func() { nodeProxyTest(f, prefix+"/nodes/", ":4194/proxy/containers/") })
 
+	// using the porter image to serve content, access the content
+	// (of multiple pods?) from multiple (endpoints/services?)
```

Comment:
this needs work. I think I know what it's doing, but I'm not 100% sure what the goal is.